dxr/testing.py

```diff
@@ -206,7 +206,8 @@ def search_results(self, query, is_case_sensitive=True):
         """
         response = self.search_response(query,
                                         is_case_sensitive=is_case_sensitive)
-        return json.loads(response.data)['results']
+        data = json.loads(response.data)
+        return data['promoted'] + data['results']
```

Comment:
I'm surprised this didn't cause more trouble.  I could see us wanting to add some functions to just return one or the other at some point, but I guess we can cross that bridge when we come to it.